src/utils/trusts.ts

```diff
@@ -181,10 +182,17 @@ const getTrustBoOthers = (
 const getIndividualTrusteesFromTrust = (
   appData: ApplicationData,
   trustId?: string,
+  isReview?: boolean,
 ): IndividualTrustee[] => {
   let individuals: IndividualTrustee[] = [];
+  let trustList: Trust[] | undefined;
+  if (isReview) {
+    trustList = appData[UpdateKey]?.[ReviewTrustKey];
+  } else {
+    trustList = appData[TrustKey];
+  }
   if (trustId) {
-    individuals = appData[TrustKey]?.find(trust =>
+    individuals = trustList?.find(trust =>
       trust?.trust_id === trustId)?.INDIVIDUALS as IndividualTrustee[];
```

Comment:
the "as IndividulTrustee[]" is not correct as result can clearly be undefined. Do we need "as"?

Reply:
this isn't newly added by Christy

Reply:
please address as tech debth